FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -308,6 +312,7 @@ const GeneralInfo = (props) => {
                     </div>
                 </form>
                 : <Loader />}
+                <ToastContainer position="top-right" autoClose={3000} />
```

Comment:
Since we always show toast messages in one place, I think it is worth to add `ToastContainer` to the page template rather than adding it for every component. We also can remove all other references of it

Reply:
ToastContainer removed from components and added to BasicPage. Also custom styles added to it, it's displayed now as per design. (Design provides styles for error notifications on Login page.) So all the notifications are styled the same way now.